components/switch/README.md

```diff
@@ -90,3 +90,21 @@ It consists of 3 states: On ("Visible"), Off ("Hidden"), and On with Conditions
 - Optional default slot content - Content that will be displayed within the "conditions" opener tooltip when the switch is on.
   
 <!-- docs: end hidden content -->
+
+### Visibility Switch with Conditions
+It may be helpful, depending on the use case, to include additional information in the label, relating to other factors that may affect the availability that the Visibility Switch represents.
+
+<!-- docs: demo code autoSize:false size:large -->
```

Comment:
Manually set the demo container `size` to `large` as that was the smallest container size where there would be enough room for the `conditions met` tooltip to show up on the bottom of the component and not to the side.